src/Service/OpenPlatform/SearchService.php

```diff
@@ -117,18 +117,21 @@ public function search(string $identifier, string $type, $refresh = false): Mate
                 throw new PlatformSearchException($exception->getMessage(), $exception->getCode());
             }
 
-            // Handle zero hit.
-            if (empty($res)) {
-                // Simply create an empty material object. Which then can be tested with
-                // the isEmpty() method.
-                $material = new Material();
-            } else {
-                $material = $this->parseResult($res);
-
-                // Check that the search IS is part of the parsed result. As this is not
-                // always the case. e.g. 9788798970804
-                if (!$material->hasIdentifier($type, $identifier)) {
-                    $material->addIdentifier($type, $identifier);
+            $material = $this->parseResult($res);
+
+            // Check that the searched for identifier is part of the parsed result. As this is not
+            // always the case. e.g. 9788798970804. This will also mean that we trust the information vendor provided
+            // information. This will also fix the issue where upload service provide a "katelog" post that we are not
+            // able to find in the datawell (doing to the way the datawell works)
```

Comment:
`doing` -> `due`

Reply:
Perhaps we should clarify further: "(Because the datawell does not allow for non-scoped search, the result we get will always be scoped to the agency credentials we search with. Materials that are not part of that agency´s collection will not be searchable)

Reply:
Added